zulip/integrations/bridge_with_slack/bridge_with_slack_config.py

```diff
@@ -3,12 +3,17 @@
         "email": "zulip-bot@email.com",
         "api_key": "put api key here",
         "site": "https://chat.zulip.org",
-        "stream": "test here",
-        "topic": "<- slack-bridge",
     },
     "slack": {
         "username": "slack_username",
         "token": "xoxb-your-slack-token",
-        "channel": "C5Z5N7R8A -- must be channel id",
+    },
+    # Mapping between Slack channels and Zulip stream-topic's.
+    # You can specify multiple pairs.
```

Comment:
I'd be inclined to give a 'multiple pair' as an example, so the format is obvious. Or maybe more than one "pairs" section, with some commented out, as alternative examples.

Then maybe even multiple examples of how it could be used. eg. multiple slack channels -> many topics in one stream, etc